backend/python/transformers-musicgen/backend.py

```diff
@@ -63,26 +63,54 @@ def LoadModel(self, request, context):
 
         return backend_pb2.Result(message="Model loaded successfully", success=True)
 
-    def TTS(self, request, context):
```

Comment:
I'd suggest let `transformers-musicgen` expose both `SoundGeneration` and `TTS`. `TTS` would accept less arguments, but helps into keeping compatibility into these cases where one want to point to a OAI-based API.

See for instance https://github.com/p0n1/epub_to_audiobook/issues/17 as a case in point

Reply:
Honestly, I've got no reason _not_ to leave the TTS side hooked up - so I'll restore that